src/browser/generate-mock.ts

```diff
@@ -0,0 +1,29 @@
+import { type Mock } from "../helpers";
+import { type GenerateForBrowserResponse } from "../node/generateForBrowser";
+
+/**
+ * Generate a list of Mock objects from GenerateForBrowserResponse function
+ * @beta
+ */
+export async function generateMock(
+    list: GenerateForBrowserResponse[],
+): Promise<Mock[]> {
+    const mockList: Mock[] = [];
+
+    for (const item of list) {
+        let { default: mock } = (await import(item.filePath.toString())) as {
+            default: Mock | { default: Mock };
```

Comment:
Jag tror du menar:

```suggestion
            Mock | { default: Mock };
```

Annars är båda typerna i union samma? Och då kommer if-satsen på rad 21 alltid vara sann (om jag går på typningen, inte verkligheten).